microcosm_flask/conventions/catchall.py

```diff
@@ -0,0 +1,27 @@
+"""
+Flask's default behavior is to return a 404 for a route the app can't match.
+For backend services that are not queried directly by a front-end it can be useful
+to specify a different HTTP code, to differentiate between a nonexistent resource
+on a well-understood path (e.g. GET /user/123) and a nonexistent (e.g. GET /ursr).
+
+In particular, this can allow a load-balancer to identify cases where it is routing a request
+based on an outdated DNS entry, and to directly retry the request on a different instance.
+
+
+"""
+from microcosm.api import binding, defaults, typed
+
+
+@binding("catchall_convention")
+@defaults(
+    fallback_http_code=typed(int, default_value=421),
```

Comment:
based on comments [here](https://github.com/globality-corp/serviceproxy/pull/170#discussion_r790676664), the default should be updated to be `501`